### PR TITLE
unify TodoService.java and TodoServiceImpl.java creating way #217

### PR DESCRIPTION
**Pull request by btyanagawamg**
Please review #217 .

The method of creating the file (TodoService.java, TodoServiceServiceImpl.java) used in REST tutorial has been changed from copying files to rewriting of the files created in Todo tutorial .

### CODE REVIEW

File: todo-rest/scripts/convert-rest-java.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+TARGET_DIR=$1
+if test -n $TARGET_DIR; then
+  pushd "$TARGET_DIR"
+fi
+
+######### TodoService.java #########
+
+# TodoService.java
+find ./ -type f -name 'TodoService.java' | xargs sed -i -e 's|Collection<Todo> findAll();|Collection<Todo> findAll();\
```
Comment (yoshikawaa) [inline todo-rest/scripts/convert-rest-java.sh]: - [ ] Wrong sed.

```java
public interface TodoService {
    Collection<Todo> findAll();

Todo findOne(String todoId);

    Todo create(Todo todo);

    Todo finish(String todoId);

    void delete(String todoId);
```

File: todo-rest/scripts/convert-rest-java.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+\
+Todo findOne(String todoId);|'
+
+
+######### TodoServiceImpl.java #########
+
+# TodoServiceImpl.java
+find ./ -type f -name 'TodoServiceImpl.java' | xargs sed -i -e 's|public Todo findOne(String todoId) {|@Transactional(readOnly = true)\
+    public Todo findOne(String todoId) {|'
```
Comment (yoshikawaa) [inline todo-rest/scripts/convert-rest-java.sh]: - [ ] You should add `@Override` annotation.

And you should change `TodoServiceImpl#findOne` method `private` to `public` at REST Tutorial (Guideline Improvement).

File: todo-rest/scripts/convert-rest-java.sh (hunk 1 above)
Comment (btyanagawamg) [reply todo-rest/scripts/convert-rest-java.sh]: cce0bd15c81af09e9d1493d2cdfd2bb9e8a53340 
Fix wrong sed indent at above commit .

File: todo-rest/scripts/convert-rest-java.sh (hunk 2 above)
Comment (btyanagawamg) [reply todo-rest/scripts/convert-rest-java.sh]: a78bc0a6993b4348576fd1d3f037bf3a4c69edd0 
In above commit, I fixed to add `@Override` annotation and change `TodoServiceImpl#findOne` method from private to public at REST tutorial .
In addition to above, I created terasolunaorg/guideline#3303 issue .

**Pull request merged**